src/main/java/com/google/sps/data/friend_graph/FriendRelationship.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.friend_graph;
+
+import java.util.Set;
+import java.util.HashSet;
+import java.lang.IllegalArgumentException;
+
+/**
+* Class to represent a single Facebook friendship between two users.
+* Represented by a pair of user IDs.
+*/
+public class FriendRelationship {
+  private String friend1ID;
```

Comment:
I can't think of an alternative at the moment, but I don't really like friend 1 and friend 2. It lacks directionality. Maybe I don't understand something.

I was under the impressions that A -> B is distinct from B -> A. If so, then the `friend1` isn't really the friend, but the "person" and `friend2` is just the "friend".

Reply:
A FriendRelationship object is meant to represent a friendship that was found from the Facebook API, and it acts sort of like an edge in the graph, and this graph should show all of the friendships found from Facebook. Because Facebook friendships are mutual, A->B would be the same as B->A.

This is part of the initial set up for finding mutual friends, and this is separate from the matches and new friends that are made through the feed.